scripts/base_network.py

```diff
@@ -560,7 +560,7 @@ def prefer_voltage(x, which):
         ~buses["under_construction"]
     )
 
-    c_nan_b = buses.country == "na"
+    c_nan_b = buses.country.isna()
```

Comment:
```suggestion
    c_nan_b = buses.country.fillna("na") == "na"
```

Reply:
did you check if this is working for multiple versions? because with some pandas versions, it also did not recognize the entries with `isna()`

Reply:
I like this (hacky) solution @FabianHofmann